### PR TITLE
Update to new Version 

### PR DESCRIPTION
**Pull request by ayush714**
Made compatible with latest version.  

Changes, 

Upgraded the Zenfile MLflow tracker to 0.7.3

### CODE REVIEW

**Comment by AlexejPenner** [comment]
Hey Ayush, what exatcly was the task for this PR - upgrading the zenfile to how mlflow is used in 0.7.3? Would be good to give a little bit of context in the PR description

**Comment by ayush714** [comment]
> Hey Ayush, what exatcly was the task for this PR - upgrading the zenfile to how mlflow is used in 0.7.3? Would be good to give a little bit of context in the PR description

Yes @AlexejPenner

**Comment by ayush714** [comment]
In `streamlit_app.py` I implemented that functionality, it will work fine but in my system I am getting some dependency error.

File: customer-satisfaction/streamlit_app.py [hunk 1]
```diff
@@ -67,9 +68,8 @@ def main():
             running=True,
         )
         if service is None:
-            # TODO [HIGH]: Print out that the service is not there, maybe run 
-            #  the pipeline first.
-            pass
+            st.write("The service is not there, Running the pipeline first to create service.")
```
Comment (AlexejPenner) [inline customer-satisfaction/streamlit_app.py:71]:
```suggestion
            st.write("No service could be found. The pipeline will be run first to create a service.")
```

**Review by AlexejPenner — CHANGES_REQUESTED** [review]
Just a tiny wording that I had a suggestion on, other than that this looks good to me

File: customer-satisfaction/streamlit_app.py (hunk 1 above)
Comment (ayush714) [reply customer-satisfaction/streamlit_app.py]: Sure, thx

**Review by AlexejPenner — APPROVED** [review]
LGTM

**Pull request merged**